lascli/__main__.py

```diff
@@ -69,7 +69,9 @@ def main():
 
     kwargs = {k: v for k, v in args.items() if v != NotProvided}
     if kwargs:
-        print(json.dumps(cmd(**kwargs), indent=2))
+        result = cmd(**kwargs)
+        result = result if kwargs.get('pretty') else json.dumps(result, indent=2)
+        print(result)
```

Comment:
Does it seem weird to care about 'pretty' here for all functions? What about refactoring a bit so that all functions are responsible for returning the str that should be output in the terminal (eg. json.dumps(...)) so that we can just use a print statement here? Or maybe event let the functions handle printing output altogether?

Reply:
I thought maybe we would expand pretty for other functions later on, and it seems to me like a nice thing to take care of the printing one place instead of having almost identical code in every function in every subparser.

Reply:
I don't like that we check for 'pretty' two places in the code (in addition to the seemingly random check of 'pretty' in main in the first place. Maybe there is a way to avoid that and also keep the printing in one place logic?